### PR TITLE
changed api endpoints for currencylayer and fixer

### PR DESCRIPTION
**Pull request by StayHomeThereIsNoFrance**

### CODE REVIEW

File: price-server/src/provider/fiat/quoter/CurrencyLayer.ts [hunk 1]
```diff
@@ -14,17 +14,18 @@ interface Response {
...
+      // access_key: this.options.apiKey,
       source: 'USD',
       currencies: this.symbols
         .map((symbol) => (symbol === 'USD/SDR' ? 'XDR' : symbol.replace('USD/', '')))
         .join(','),
     }
 
     const response: Response = await fetch(
-      `https://apilayer.net/api/live?${toQueryString(params)}`,
+      `https://api.apilayer.com/currency_data/live?${toQueryString(params)}`,
```
Comment (PFC-developer) [inline price-server/src/provider/fiat/quoter/CurrencyLayer.ts:25]: are you sure?

https://apilayer.net/api/live?access_key=XXX&currencies=EUR,GBP,CAD,PLN&source=USD&format=1

works

 curl -i 'https://api.apilayer.com/currency_data/live?access_key=XXXX&currencies=EUR,GBP,CAD,PLN&source=USD&
format=1'

gives a 401 error

Comment (clemensgg) [reply price-server/src/provider/fiat/quoter/CurrencyLayer.ts:25]: i was able to get it working with this PR with our newly registered currencylayer & fixer apis, but not with 'https://apilayer.net/api/live?access_key=XXX&currencies=EUR,GBP,CAD,PLN&source=USD&format=1' 

seems like newer plans that are purchased via apilayer are only working with the new endpoint while old ones don't? that would be a real api mess

**Comment by hanjukim** [comment]
Can we exclude changes in currencylayer? It seems that new endpoint is not documented yet.

File: price-server/src/provider/fiat/quoter/CurrencyLayer.ts (hunk 1 above)
Comment (frankfka) [reply price-server/src/provider/fiat/quoter/CurrencyLayer.ts:25]: What about `https://api.currencylayer.com/live?access_key=xxx&currencies=AUD,EUR,GBP,PLN&source=USD&`? This works for my old API key, and is reflected in the [docs](https://currencylayer.com/documentation)

**Comment by etienne-napoleone** [comment]
maybe it should be a different provider, it still need to support ppl running old enpoints

**Comment by ilhanu** [comment]
This PR changes work for us, just implemented it. Currencylayer renamed itsself for new customer to apilayer. 
The documentation is here : https://apilayer.com/marketplace/description/currency_data-api

File: price-server/src/provider/fiat/quoter/CurrencyLayer.ts (hunk 1 above)
Comment (StayHomeThereIsNoFrance) [reply price-server/src/provider/fiat/quoter/CurrencyLayer.ts:25]: > are you sure?
> 
> https://apilayer.net/api/live?access_key=XXX&currencies=EUR,GBP,CAD,PLN&source=USD&format=1
> 
> works
> 
> curl -i 'https://api.apilayer.com/currency_data/live?access_key=XXXX&currencies=EUR,GBP,CAD,PLN&source=USD& format=1'
> 
> gives a 401 error

from their docs you should use 

```
curl --request GET 'https://api.apilayer.com/currency_data/live?base=USD&symbols=EUR,GBP' \
--header 'apikey: YOUR API KEY'
```

old URLs don't work with newly registered accounts

**Comment by emidev98** [comment]
Hey all, I will proceed to close the pull request because seems to be an old one and the oracle feeder will change its propose. But please feel free to create new pull request with the new code basis, thanks you!

**Pull request closed without merging**